src/commands/synthetics/utils.ts

```diff
@@ -829,23 +909,6 @@ export const getResultUrl = (baseUrl: string, test: Test, resultId: string) => {
   return `${testDetailUrl}?resultId=${resultId}&${ciQueryParam}`
 }
 
-/**
- * Sort results with the following rules:
- * - Passed results come first
- * - Then non-blocking failed results
- * - And finally failed results
- */
```

Comment:
This was used for rendering results in a specific order before #1097. Since we now render results **as they arrive**, we can remove this function.